libs/langchain/langchain/schema/runnable/passthrough.py

```diff
@@ -38,6 +56,30 @@ def InputType(self) -> Any:
     def OutputType(self) -> Any:
         return self.input_type or Any
 
+    @classmethod
+    def assign(
```

Comment:
Should we add an `assign` method to RunnableMap?

Reply:
What would we use that for?